File: CRM/Models/ViewModels/LeadAssignmentRatingVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CRM.Models.ViewModels
{
    public class LeadAssignmentRatingVM : IAssessable
    {
        public Guid LeadId { get; set; }
        public int LeadAssignmentId { get; set; }

        public int Rate { get; set; }
        public string Comment { get; set; }
        public DateTime CommentedOn { get; set; }
        public string CommentedBy { get; set; }
    }
}

File: CRM/wwwroot/js/viewmodels/LeadAssignmentRatingVM.js
class LeadAssignmentRatingVM {
    constructor(leadId, leadAssignmentId, comment) {
        this.LeadId = leadId;
        this.LeadAssignmentId = leadAssignmentId;
        this.Comment = comment;
        this.Rate = 0;
        this.CommentedOn = new Date();
        this.CommentedBy = "user";
    }
}